tensorflow/core/grappler/optimizers/remapper_test.cc

```diff
@@ -984,7 +974,7 @@ TEST_F(RemapperTest, FuseConv2DWithSqueezeAndBias) {
   ASSERT_EQ(tensors.size(), 1);
   test::ExpectTensorNear<float>(tensors[0], tensors_expected[0], 1e-6);
 }
-#endif
+#endif  // !INTEL_MKL
```

Comment:
So this means `FuseConv2DWithBatchNorm`, `FuseConv2DWithBatchNormAndActivation`, and `FuseConv2DWithSqueezeAndBias` tests will remain disabled for MKL build, right?

Reply:
Yes, these fusions are not supported by MKL currently.